doc/lua_api.txt

```diff
@@ -2284,6 +2363,44 @@ reserved to pass key press events to formspec!
 
 
 
+Migrating to Real Coordinates
+=============================
```

Comment:
Make this be a 2nd level heading?

Reply:
oops. Yeah, this should be a heading below the formspec one